<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temporal Order Judgement Experiment</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="globalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="optionsLayout">
       <item>
        <widget class="QGroupBox" name="whichCameFirst">
         <property name="title">
          <string>Which came first?</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="rbDot">
            <property name="text">
             <string>Dot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbTone">
            <property name="text">
             <string>Tone</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="howConfident">
         <property name="title">
          <string>How confident are you of this answer?</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rbConf5">
            <property name="text">
             <string>Very confident</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbConf4">
            <property name="text">
             <string>Somewhat confident</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbConf3">
            <property name="text">
             <string>Neither confident nor unconfident</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbConf2">
            <property name="text">
             <string>Somewhat unconfident</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbConf1">
            <property name="text">
             <string>Very unconfident</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="okButton">
         <item>
          <widget class="QPushButton" name="btOk">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
